File: python/Iniciante/URI1036.py
from math import sqrt

linha = input().split()

a, b, c = list(map(float, linha))

if a == 0 or b * b - 4 * a * c < 0:
  print("Impossivel calcular")
else:
  bascara = sqrt(b * b - 4 * a * c)
  r1 = ((b*-1) + bascara) / (2 * a)
  r2 = ((b*-1) - bascara) / (2 * a)

  print(f"R1 = {r1:.5f}")
  print(f"R2 = {r2:.5f}")

File: java/iniciante/URI1036.java
package URI;

import java.util.Scanner;

public class URI1036 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double A = scanner.nextDouble(), B = scanner.nextDouble(), C = scanner.nextDouble();

        if (Math.pow(B, 2) - 4 * A * C > 0) {
            double bask = (B * B) - 4 * A * C;
            double r1 = (-B + Math.sqrt(bask)) / (2 * A);
            double r2 = (-B - Math.sqrt(bask)) / (2 * A);
            System.out.printf("R1 = %.5f\n", r1);
            System.out.printf("R2 = %.5f\n", r2);
        } else {
            System.out.println("Impossivel calcular");
        }
    }
}
